kolibri/content/serializers.py

```diff
@@ -378,15 +385,21 @@ def get_on_device_resources(self, instance):
             .count()
 
     def get_num_coach_contents(self, instance):
-        # TODO return 0 by default if user is logged in as a Learner
-        if instance.kind == content_kinds.TOPIC:
-            return instance.get_descendants() \
-                .filter(coach_content=True, available=True) \
-                .exclude(kind=content_kinds.TOPIC) \
-                .distinct() \
-                .count()
-        else:
-            return 1 if instance.coach_content else 0
+        user = self.context["request"].user
```

Comment:
I think having this code here too is fine, but this particular Serializer is only used in the Device Management Page, so only Admins would be calling this particular endpoint.

I'm hoping in the future we could somehow consolidate `ContentNode` and `ContentNodeGranular` serializers.

Reply:
ahh yes thats true. I'll remove it to not create any confusion